src/components/hero-section.js

```diff
@@ -45,6 +45,7 @@ const ButtonWrapper = styled.div`
 
   .btn {
     box-shadow: 0 1.25rem 2.5rem #021717;
+    font-size: xx-large;
```

Comment:
Please review this change on laptop and mobile devices. The text is way too big for smaller screens. You should move this `font-size: xx-large` rule inside of a media query so that the text only becomes xx-large on desktop screen sizes.

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/50115852/125873076-94224080-f04e-42f0-ae7b-036f1f93d475.png">

<img width="503" alt="image" src="https://user-images.githubusercontent.com/50115852/125873283-5d4b511f-3e1c-49c2-bc78-4b8a164a04a1.png">